src/data_processing/trying_again.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import os
+
+
+path = os.getcwd()
+path = os.path.abspath(os.path.join(path, os.pardir))
```

Comment:
The two `path = os.path.abspath(os.path.join(path, os.pardir))` are only necessary if you have cd into this folder when running the cell. Either mention that you have to cd in the folder to run this file or take this two lines out